torch/fx/experimental/proxy_tensor.py

```diff
@@ -801,6 +801,10 @@ def can_handle_tensor(x: Tensor) -> bool:
             if r is not NotImplemented:
                 return r
 
+    if func is torch.ops.aten.is_nonzero.default:
+        with proxy_mode:
+            return (args[0] != 0).item()  # type: ignore[attr-defined]
```

Comment:
This doesn't do anything anymore right?

Reply:
I guess this still enables export explicit is_nonzero calls in non-strict + real-tensor mode